db/re/item_db_equip.yml

```diff
@@ -98932,7 +98928,6 @@ Body:
     Refineable: true
     Script: |
       .@r = getrefine();
-      bonus bMaxHP,.@r*10;
```

Comment:
The bonus is here on aegis data but yeah it's missing on the description.. don't remove it

Reply:
Thanks